index.html

```diff
@@ -0,0 +1,245 @@
+                        <p class="date-p px-3 py-4 col-md-10 col-md-start">@ National Museum of Karea, Art Center Nabi and more
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- main program -->
+    <section id="main-program">
+        <div class="container-lg">
+            <div class="text-center pt-5">
+                <h2 class="main-program">Main Program</h2>
+                <div class="line"></div>
+            </div>
+            <div class="row my-5 align-items-center justify-content-center g-1">
+
+                <div class="col-md-2 program-card">
+                    <div class="card border-0">
+                        <div class="card-body text-center">
+                            <img src="assets/main-program/program_icon_01.png" alt="program icon" class="img-fluid mb-4">
+                            <p class="card-subtitle">Lecture</p>
+                        </div>
+                        <div class="card-body">    
+                            <p class="card-text text-center">Listen to the speakers from various countries about the messages of sharing and opening.</p>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="col-md-2 program-card">
+                    <div class="card border-0">
+                        <div class="card-body text-center">
+                            <img src="assets/main-program/program_icon_02.png" alt="Program icon" class="img-fluid mb-4">
+                            <p class="card-subtitle">CC Exhibition</p>
+                        </div>
+                        <div class="card-body">    
+                            <p class="card-text text-center">Appreciate various creations applying CC license of artists, organized from Art</p>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="col-md-2 program-card">
+                    <div class="card border-0">
+                        <div class="card-body text-center">
+                            <img src="assets/main-program/program_icon_03.png" alt="program icon" class="img-fluid mb-4">
+                            <p class="card-subtitle">Forum</p>
+                        </div>
+                        <div class="card-body">    
+                            <p class="card-text text-center">Have the time to share your thoughts and opinions with experts for each topic.</p>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="col-md-2 program-card">
+                    <div class="card border-0">
+                        <div class="card-body text-center">
+                            <img src="assets/main-program/program_icon_04.png" alt="Program icon" class="img-fluid mb-4">
```

Comment:
- [ ] Kindly ensure that description text of each card in the **Main program** section on mobile is correctly aligned as now it looks different with the original deisign

| Template Design | Your Design |
| ---------------------- | ----------------- |
| ![Screenshot from 2023-07-06 19-33-42](https://github.com/MozamelJawad/Capstone-Project/assets/83042742/e8ca2464-e449-4e5f-948a-46d327b1ed5e) | ![Screenshot from 2023-07-06 19-34-17](https://github.com/MozamelJawad/Capstone-Project/assets/83042742/8edf89a9-f68e-4d1e-9e8c-d43cfef60767) |